src/types/config.js

```diff
@@ -54,13 +54,33 @@
  */
 
 /**
- * @typedef RequestConfigFromFrameOptions
+ * May be provided by `AnnotatorConfigFromHost` to configure a known ancestor
+ * frame as the "embedder" frame. `ConfigFromHost` will be requested from this
+ * frame, and additional messaging (via JSON RPC) may be configured between
```

Comment:
```suggestion
 * frame, and additional messaging (via `postMessage`) may be configured between
```

I think the `postMessage` aspect of the transport is the more important part here. The JSON-RPC protocol could be swapped without affecting the overall picture.